packages/components/src/components/gradient-background-two-point/gradient-background-two-point.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { CSSProperties } from 'react';
+import './gradient-background-two-point.scss';
+
+type TGradientBackgroundTwoPoint = {
+    blurRadius?: number;
```

Comment:
@sergei-deriv @mahdiyeh-deriv according to our naming convention, snake_case should be used if it's not a function. Applies to all variables in this file.